python/test/compose_test.py

```diff
@@ -0,0 +1,99 @@
+""" cloudstorage compose tester"""
+import webapp2
+import logging
+import re
+import cloudstorage
+from google.appengine.api import app_identity
```

Comment:
Did you speak with John. I think this may still be an issue and would need to change to:

try:
  from cloudstorage import errors
except ImportError:
  from google.appengine.ext.cloudstorage import errors

Like in cloudstorage_test.py